pages/connections/farmlands.vue

```diff
@@ -103,9 +103,28 @@ const externalNavigation = [
 ];
 
 useHead({
-  title: 'Connections - Farmlands',
+  title: 'Farmlands - Centrapay Docs',
   meta: [
-    { name: 'Farmlands Merchants - How to integrate with Centrapay', description: 'Farmlands has partnered with Centrapay to provide new ways for your business to accept Farmlands Cards faster, easier and more securely than ever.' }
+    {
+      name: 'description',
+      content: 'Farmlands has partnered with Centrapay to provide new ways for your business to accept Farmlands Cards faster, easier and more securely than ever.',
+    },
+    {
+      property: 'og:image',
+      content: '/countryside.jpg',
+    },
+    {
+      property: 'og:image:width',
+      content: '1200'
+    },
+    {
+      property: 'og:image:height',
+      content: '630'
```

Comment:
This is the recommended image size for OG - https://www.kapwing.com/resources/what-is-an-og-image-make-and-format-og-images-for-your-blog-or-webpage/#:~:text=OG%20Images%20are%20the%20first,ratio%20of%201.91%3A1).